projectDetails.txt

```diff
@@ -0,0 +1,56 @@
+Project Name:
+    "NYT Reading List"
+
+Elevator Pitch:
+    The who:
+    Users hoping to create a reading list with various resources available through the NYT.
+
+    The value:
+    - Being able to conveniently search a large number of channels within the NYT media universe.
+    - Being able to create a personal reading list without having to create a NYT account.
+
+
+Dependencies:
+    - NYT API (https://developer.nytimes.com/apis) + fetch API
```

Comment:
Note that the API key for the NYT isn't supposed to be exposed in a React app. It's recommended to call that API from a web service. But if you don't have time to do that, it's fine. Just keep in mind it could be deactivated if someone uses/abuses the API with the key.